spec/lib/fasterer/file_traverser_spec.rb

```diff
@@ -334,4 +334,24 @@
         .to match_array(['user.rb - RubyParser::SyntaxError - unterminated string meets end of file. near line 1: ""'])
     end
   end
+
+  describe 'output' do
+    let(:test_file_path) {RSpec.root.join('support', 'output', 'sample_code.rb')}
```

Comment:
If you ran rubocop, it would state that the opening bracket should have a space after, and a closing one a space before.